pkg/cluster/internal/providers/podman/provision.go

```diff
@@ -206,6 +206,11 @@ func runArgsForNode(node *config.Node, clusterIPFamily config.ClusterIPFamily, n
 		"--volume", fmt.Sprintf("%s:/var:suid,exec,dev", varVolume),
 		// some k8s things want to read /lib/modules
 		"--volume", "/lib/modules:/lib/modules:ro",
+		// propagate KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER to the entrypoint script
+		"-e", "KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER",
+		// enable /dev/fuse explicitly for fuse-overlayfs
+		// (Rootless Podman does not automatically mount /dev/fuse with --privileged)
```

Comment:
What's the reason for that? I'm only familiar with the rough concept of fuse 😅

Reply:
Because `fuse-overlayfs` needs `/dev/fuse` to mount FUSE.

Reply:
> Because `fuse-overlayfs` needs `/dev/fuse` to mount FUSE.

Sorry, that part I get, what I meant ways: What's the reason for (the comment):

> Rootless Podman does not automatically mount /dev/fuse with --privileged

Why this deviation?

Reply:
Kinda bug / TODO (of runc) due to complexity

Reply:
thanks!

Reply:
it sounds like they don't because it may not be present see e.g. https://github.com/openshift/enhancements/issues/362